### ISSUE TITLE
[Azure Toolkit][A11yMAS][Accessibility][Inspect][MAS4.2.1][Eclipse][Wipro]: Name property is empty for document in "Your Subscriptions dialog"

### ISSUE THREAD

**Issue by WiproPSAccTest** [Microsoft/azure-tools-for-java#1392]
**Repro Steps:**

1. Open and Start Eclipse.
2. Navigate to Tools menu-> "Azure"-> "Azure Sign In"
3. In "Azure Sign in" dialog, tab to and select "Automated" radio button.
4. Tab to and select "Browse" button.
5. In "Create Authentication Files" dialog , select the required subscriptions and select "Start" button.
6. In "Service Principal Creation Status dialog", tab to and select on button/.
7. In "Your Subscriptions" dialog , tab to document and observe the name property using inspect.

**Actual Result:**
Name property is empty for the document..
Narrator is not announcing the value.value information for the document.

**Narrator behaviour:**
Tab

**NVDA behaviour:**
edit  read only
blank

**Expected Result:**
Name property should be simple and meaningful.
Name property for the document should be "Your Subscriptions".
Narrator should  announce the value.value for the document .

**Platform:**
Eclipse Version:
Eclipse Java EE IDE for Web Developers.
Version: Oxygen.2 Release (4.7.2)
Build id: 20171218-0600
OSBuild Version: Version 1709(Os Build 16299.192)
![your subscriptions-document-no name](https://user-images.githubusercontent.com/22956130/35682875-336d806a-0788-11e8-88a3-545073b8b81f.jpg)
![your subscriptions-document-nvda](https://user-images.githubusercontent.com/22956130/35682879-3688165c-0788-11e8-999e-7cf012250fe5.jpg)

**Comment by Eskibear** [comment]
#1407

**Comment by Eskibear** [comment]
tracked in https://bugs.eclipse.org/bugs/show_bug.cgi?id=533976

**Comment by Eskibear** [comment]
* **Importance of experience**: minimal. Very low usage based on telemetry
* **Quality of experience**: not impacted. Users can still see the property in the `Value` field
* **Alternate solutions**: No alternate technologies or options can be leveraged.
* **Evidence**: All evidence is provided in this issue and external issue.

![image](https://user-images.githubusercontent.com/2351748/46598123-84f1f000-cb15-11e8-88eb-3a962b0f95aa.png)

**Comment by jdneo** [comment]
Can show the name property on my side.

Eclipse Version: Photon Release (4.8.0)
Plugin Version: 3.11.0

![image](https://user-images.githubusercontent.com/6193897/47150337-ddd93980-d308-11e8-9d85-2a01ac237421.png)

**Comment by Tanushagottipati** [comment]
This issue is still reproducing in the current test pass

**Comment by vscjavaci** [comment]
Track internal issue by AB#1749845
